jsonld-contexts/consent.jsonld

```diff
@@ -0,0 +1,10 @@
+{
+	"@context": {
+		"UserConsent": "https://vocab.egm.io/UserConsent",
+		"provider": "https://vocab.egm.io/provider",
+		"validityPeriod": "https://vocab.egm.io/validityPeriod",
+		"startAt": "https://vocab.egm.io/startAt",
```

Comment:
`startAt` and `endAt` are in the core context

Reply:
Also better avoid to use terms from the core context (outside of what they are designed for, of course). In this case, we have to invent two new terms instead of relying on the core context. Maybe startsAt and endsAt ?